main.py

```diff
@@ -1,71 +1,45 @@
-import uuid
-
-from fastapi import Depends
-from fastapi import FastAPI, Form
-from fastapi import Request, Response
-from fastapi.responses import HTMLResponse
-from fastapi.templating import Jinja2Templates
-from sqlalchemy.orm import Session
+from sqlalchemy import Integer, String, Text
+from sqlalchemy.orm import Session, Mapped, mapped_column
 
 from database import Base
-from database import SessionLocal
-from database import engine
-from models import create_todo
-from models import delete_todo
-from models import get_todo
-from models import get_todos
-from models import update_todo
-
-Base.metadata.create_all(bind=engine)
 
-app = FastAPI()
-templates = Jinja2Templates(directory="templates")
 
+class ToDo(Base):
+    __tablename__ = "todos"
 
-def get_db():
-    db = SessionLocal()
-    try:
-        yield db
-    finally:
-        db.close()
+    id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
+    content: Mapped[str] = mapped_column(String)
+    session_key: Mapped[str] = mapped_column(String)
+    notes: Mapped[str | None] = mapped_column(Text, nullable=True)
 
 
-@app.get("/", response_class=HTMLResponse)
-def home(request: Request, db: Session = Depends(get_db)):
-    session_key = request.cookies.get("session_key", uuid.uuid4().hex)
-    todos = get_todos(db, session_key)
-    context = {
```

Comment:
Consider adding error handling for cases where the todo item doesn't exist when trying to delete.